cookiecutter/{{cookiecutter.github_repo_name}}/CONTRIBUTING.md

```diff
@@ -0,0 +1,101 @@
+# Contributing to {{cookiecutter.title}}
+
```

Comment:
I'd suggest adding running `protolint` as a recommended step.

Reply:
OK, the issue with `protolint` is that it is *difficult* to install (not present in Ubuntu basically), but we could point to the `protolint` install instructions and let the user deal with it.

Reply:
>but we could point to the protolint install instructions and let the user deal with it.

Agreed.

Reply:
As said in the other comment, I would prefer to first bring `repo-config` on par to the status quo so I can merge this PR and get it to a more usable state. For example I also have a separate issue to add a CI check using `protolint`:

* https://github.com/frequenz-floss/frequenz-repo-config-python/issues/32

Ideally `protolint` should be run by `nox` (but because it is hard to install I was thinking of maybe making it optional). In that case these instructions will get obsoleted soon. Anyway, the `protolint` needs a bit more thought IMHO.

Would it be OK to add a comment to the `protolint` issue saying that when we add it to the CI we should also provide instructions (or an easy way) for the user run it locally?

Reply:
> Ideally protolint should be run by nox

Why?
I'd not suggest mixing up the protobuf and python toolchains. IMO it will introduce unnecessary coupling between the API specs and the auxiliary python library.

> Would it be OK to add a comment to the protolint issue saying that when we add it to the CI we should also provide instructions (or an easy way) for the user run it locally?

Is it worth waiting for an issue for adding a couple lines?

Reply:
> I'd not suggest mixing up the protobuf and python toolchains. IMO it will introduce unnecessary coupling between the API specs and the auxiliary python library.

OK, good point, I guess then the API repos are a bit of an alien here, because this `repo-config` is super focused on Python (the repo name is `frequenz-repo-config-python`).

In any case, my long term plan is to completely ditch `nox` (for other reasons). I guess for practical reasons we should still keep a repo type "api" in here, but in some future maybe we want to change that and have this repo only take care of the Python side of the "api" repos.

> Is it worth waiting for an issue for adding a couple lines?

To do it proper, is not just adding a couple of lines, ideally I should test it and it is a distraction as it is not what I wanted to focus on right now, but I guess it will still be quicker to add it rather than keep discussing it here :)

Reply:
Done.

Reply:
>but in some future maybe we want to change that and have this repo only take care of the Python side of the "api" repos.

Makes sense.

>To do it proper, is not just adding a couple of lines, ideally I should test it and it is a distraction as it is not what I wanted to focus on right now,

Ok understood. Thanks anyway for adding it.